# HG changeset patch
# User Rik <rik@octave.org>
# Date 1478754248 28800
#      Wed Nov 09 21:04:08 2016 -0800
# Node ID 9506842e0f6fda3d70bb9be85411c24a32a5f7e6
# Parent  2df51376f58712838a0b34ba41cac6555c2c3329
odedefaults.m: Don't require vector for JPattern, MvPattern (bug #49564).

* odedefaults.m: Remove "vector" from attributes cell array for JPattern and
MvPattern later used by validateattributes().

diff --git a/scripts/ode/private/odedefaults.m b/scripts/ode/private/odedefaults.m
--- a/scripts/ode/private/odedefaults.m
+++ b/scripts/ode/private/odedefaults.m
@@ -75,23 +75,23 @@ function [defaults, classes, attributes]
 
   persistent attributes = struct ("AbsTol", {{"real", "vector", "positive"}},
                                   "BDF", {{"on", "off"}},
                                   "Events", {{}},
                                   "InitialSlope", {{"real", "vector", "numel", n}},
                                   "InitialStep", {{"positive", "scalar"}},
                                   "Jacobian", {{}},
                                   "JConstant", {{"on", "off"}},
-                                  "JPattern", {{"vector"}},
+                                  "JPattern", {{}},
                                   "Mass", {{}},
                                   "MassSingular", {{"no", "maybe", "yes"}},
                                   "MaxOrder", {{">=", 0, "<=", 5, "integer"}},
                                   "MaxStep", {{"positive", "scalar", "real"}},
                                   "MStateDependence", {{"weak", "strong", "none"}},
-                                  "MvPattern", {{"vector"}},
+                                  "MvPattern", {{}},
                                   "NonNegative", {{"vector", "integer", "positive"}},
                                   "NormControl", {{"on", "off"}},
                                   "OutputFcn", {{}},
                                   "OutputSel", {{"vector", "integer", "positive",...
                                                  ">", 0, "<=", n}},
                                   "Refine", {{"scalar", ">", 0, "integer"}},
                                   "RelTol", {{"scalar", "positive", "real"}},
                                   "Stats", {{"on", "off"}},
